.github/workflows/locale.yml

```diff
@@ -290,3 +290,24 @@ jobs:
         run: |
           make check
           if grep "DEPRECATED" Testing/Temporary/LastTest.log; then exit 1; fi
+  linux-da-dk:
+    name: "linux, da-dk"
+    runs-on: "ubuntu-20.04"
+    env:
+      LANG: "da_dk.UTF-8"
```

Comment:
I don't think the casing will cause anything to break, but best to change this to `da_DK.UTF-8` for consistency with the other tests.